patches/chrome-common-webui_url_constants.cc.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/chrome/common/webui_url_constants.cc b/chrome/common/webui_url_constants.cc
+index 3a720f10d3ac49699639889e1f16cb75f9057c89..acabe934cb6c59c340feb23acc8bd08c51edfc38 100644
+--- a/chrome/common/webui_url_constants.cc
++++ b/chrome/common/webui_url_constants.cc
+@@ -611,6 +611,7 @@ const char* const kChromeHostURLs[] = {
+     kChromeUIDevicesHost,
+ #endif
+     kChromeUIWebRtcLogsHost,
++BRAVE_HOST_URLS
```

Comment:
Since the placing of this line is irrelevant for the runtime, I'd suggest putting it right on the first line so we minimize any possible chrome conflicts in the future.
```
const char* const kChromeHostURLs[] = {
    BRAVE_HOST_URLS
    kChromeUIAboutHost,
```

Reply:
but actually, we can substitute our hosts by redefining any of ```content::``` hosts and don't introduce the patch at all.

Reply:
++ on this; this was the cause of the failed patch 😄

Reply:
Hope you don't mind @nullhook - I just rebased this branch and then moved the line to the top 😄 Will give this a go

Reply:
@bsclifton what about removing the patch?
> we can substitute our hosts by redefining any of ```content::``` hosts and don't introduce the patch at all.
```
#include "chrome/common/webui_url_constants.h"

#define kChromeUIAppCacheInternalsHost \
    kChromeUIAppCacheInternalsHost,
    BRAVE_HOST_URLS

...
```

Reply:
@nullhook ^^
> what about removing the patch?

Reply:
@goodov which page would you suggest we substitute? It should be one that doesn't work / isn't used

If `brave://memories` is in the content:: area, that would be a great way to close out two issues in one go

Reply:
why it should be the one that's not used?
I'm not talking about replacing the content of some variable. I'm talking about using ```#define``` for some a variable that is not defined inside ```webui_url_constants.h```.

Reply:
such a clever override!

> the fact that we have content::  variables declared in other file allowed this trick. otherwise the patch is the only way

Reply:
Woah - nice! Now I see what you mean. Good call, @goodov 😄